src/lib/openapi/spec/create-feature-strategy-schema.ts

```diff
@@ -22,6 +22,13 @@ export const createFeatureStrategySchema = {
         parameters: {
             $ref: '#/components/schemas/parametersSchema',
         },
+        segments: {
+            type: 'array',
+            description: 'Ids of segments to use for this strategy',
+            items: {
+                type: 'number',
+            },
+        },
```

Comment:
This seems to be missing on the schema, I appreciate an extra pair of eyes to validate this. It's used when adding a strategy to a toggle from the admin.